ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/domain-lookup.j2

```diff
@@ -1,11 +1,11 @@
 {# eos - domain lookup #}
 {% if ip_domain_lookup is arista.avd.defined %}
-{%     for source in ip_domain_lookup.source_interfaces | arista.avd.natural_sort %}
+{%     for source in ip_domain_lookup.source_interfaces | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
 {%         set ip_domain_cli = "ip domain lookup" %}
```

Comment:
I think we need to add an if check for ```source.name``` is avd.defined. Because natural_sort won't be handling if the variable is defined or not in case of list of dictionaries.

Reply:
Although, convert_dicts will ensure the name variable is created, IMO it's safe to do an if check before using it (in line 8) because this dictionary to list conversion might be changed anytime in future. Any comments @ClausHolbechArista